catalogs/README.rst

```diff
@@ -6,12 +6,23 @@ This means we don't rely on other websites and data sources to be stable.
 
 The licences and references are:
 
-- 1LHAASO:
+- The first LHAASO catalog (1LHAASO):
     - Reference: `Zhen Cao et al. 2024 ApJS 271 25 <https://iopscience.iop.org/article/10.3847/1538-4365/acfd29>`_
     - Licence: The First LHAASO Catalog of Gamma-Ray Sources © 2023 by The LHAASO
       Collaboration is licensed under `CC BY-NC-SA 4.0 <https://creativecommons.org/licenses/by-nc-sa/4.0/>`_.
-- 2HWC:
-- 3HWC:
-- HGPS:
+- The Second HAWC Catalog of Very-high-energy Gamma-Ray Sources (2HWC):
+    - Reference: `A. U. Abeysekara et al., 2017, ApJ 843 40 <https://iopscience.iop.org/article/10.3847/1538-4357/aa7556>`_
+    - Licence: adapted and modified from the HAWC public datasets (see `this link <https://data.hawc-observatory.org/about.php>`_),
+      these data are licenced like the other Gammapy data, ie under
+      `CC BY-NC-SA 4.0 <https://creativecommons.org/licenses/by-nc-sa/4.0/>`_.
+- The Third HAWC Catalog of Very-high-energy Gamma-Ray Sources (3HWC):
+    - Reference: `A. Albert et al., 2020, ApJ 905 76 <https://iopscience.iop.org/article/10.3847/1538-4357/abc2d8>`_
+    - Licence: adapted and modified from the HAWC public datasets (see `this link <https://data.hawc-observatory.org/about.php>`_),
+      these data are licenced like the other Gammapy data, ie under
+      `CC BY-NC-SA 4.0 <https://creativecommons.org/licenses/by-nc-sa/4.0/>`_.
+- The H.E.S.S. Galactic Plane catalogue (HGPS):
    - Reference: `H.E.S.S. Coll, A&A 612, A1 (2018) <https://www.aanda.org/articles/aa/full_html/2018/04/aa32098-17/aa32098-17.html>`_
-   - Licence: H.E.S.S. Galactic Plane catalogue © 2018 by the H.E.S.S. Collaboration is licensed under `CC BY 4.0 <https://creativecommons.org/licenses/by/4.0/>`_.
\ No newline at end of file
+   - Licence: H.E.S.S. Galactic Plane catalogue © 2018 by the H.E.S.S. Collaboration is licensed under `CC BY 4.0 <https://creativecommons.org/licenses/by/4.0/>`_.
```

Comment:
Why is HESS under CC-4.0 and not CC BY-NC-SA 4.0 ?

Reply:
The used file is directly from the HESS collaboration. And the HESS management told me to use this licence. One can not use an other licence